app/models/course/assessment/category.rb

```diff
@@ -8,10 +10,16 @@ class Course::Assessment::Category < ActiveRecord::Base
 
   accepts_nested_attributes_for :tabs
 
+  before_create :build_initial_tab
   before_destroy :validate_before_destroy
 
   default_scope { order(:weight) }
 
+  def self.after_course_create(course)
+    course.assessment_categories.
+      create(title: Course::Assessment::Category.human_attribute_name('title.default'), weight: 0)
```

Comment:
which means we're talking about moving this key out to some other scope?

I'm more concerned if when creating a course we might use the wrong localisation, e.g. Chinese user creates a new course, he'd expect the Chinese translation of "Assessments"